docs/source/conf.py

```diff
@@ -21,39 +25,60 @@
 author = 'deepchem-contributors'
 
 # The full version, including alpha/beta/rc tags
-release = '2.4.0rc'
+version = deepchem.__version__
+release = deepchem.__version__
 
 # -- General configuration ---------------------------------------------------
 
-master_doc = 'index'
-
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'sphinx.ext.autodoc', 'sphinx.ext.autosummary', 'sphinx.ext.doctest',
-    'sphinx.ext.intersphinx', 'sphinx.ext.mathjax', 'sphinx.ext.ifconfig',
-    'sphinx.ext.napoleon'
+    'sphinx.ext.autodoc',
+    'sphinx.ext.napoleon',
+    'sphinx.ext.doctest',
+    'sphinx.ext.linkcode',
+    'sphinx.ext.mathjax',
+    'sphinx.ext.autosectionlabel',
 ]
 
-autosummary_generate = True
-autodoc_default_flags = ['members', 'inherited-members']
-numpydoc_class_members_toctree = False
```

Comment:
- `autosummary_generate` is not used in docs.
- `autodoc_default_flags ` is the deprecated api.
- `numpydoc_class_members_toctree ` is not needed because we don't use `numpydoc` extension.

Reply:
Do we need `numpydock_class_members_toctree` to handle the fact that our docstrings are written in numpy style? If we don't need it, good to remove!

Reply:
`numpydock_class_members_toctree`  is only needed by using `numpydoc` extension. But, the current docs use `sphinx.ext.napoleon` for parsing Numpy style docstring instead of `numpydoc`. So, I think this option is not valid, not necessary.